locales/translation-template.json

```diff
@@ -79,6 +79,10 @@
     "defaultMessage": "Feedback Sent",
     "description": "Feedback Sent"
   },
+  "filterByTags": {
```

Comment:
I see these changes are not related not the jira card. Are these just additional fixes?

Reply:
@Hyperkid123 They appeared when I ran `npm run translations`  Should I remove them?